_config.yml

```diff
@@ -3,4 +3,4 @@ title: Reciprocal Space Station
 plugins:
   - jekyll-font-awesome-sass
 
-port: 4503
```

Comment:
Note: this parameter is nonsense related to my local builds of the site, and has no bearing on anything else.

Reply:
can you delete this line? if yes, do so -- it's something that someone may trip over later

if not, add a comment before merging this PR